### PR TITLE
Fix typos in jira plugin description

### LINKED ISSUE #109: Fix JIRA description
Hi,

There is some typo in the [jira plugin page](https://roadie.io/backstage/plugins/jira/)

##### 1. Typo in the import statement

image

Missing a coma after `JiraCard`

##### 2. Typo in the `X-Atlassian-Token` header

The value should be `nocheck` without a dash, per this anwser: https://community.atlassian.com/t5/Answers-Developer-Questions/Getting-past-XSRF-check-failures-when-attempting-to-post-to-Rest/qaq-p/522137#M59793

##### 3. csp policy

To access images from Jira, the backstage csp should be changes to allow `*.atlassian.net` (or the jira instance url) in `backend.csp.img-src`

---

Would you accept a PR to fix theses in [`content/plugins/descriptions/jira.yaml`](https://github.com/RoadieHQ/marketing-site/blob/3f26d0a31f7eea8bad16ab8694f042f81e12c983/content/plugins/descriptions/jira.yaml) ?

### PR DESCRIPTION
**Pull request by rbideau**
- Fix missing coma in import statement
- Remove dash X-Atlassian-Token header value
- Add instruction for csp

Fixes #109

### CODE REVIEW

**Review by dtuite — APPROVED** [review]

**Comment by dtuite** [comment]
Thanks @rbideau. You should be able to merge this yourself when ready. It will deploy automatically.

**Comment by rbideau** [comment]
Thanks @dtuite.

The PR is ready but I don't have write access, I think only you can merge.

**Pull request merged**
